src/test/rustdoc-ui/intra-links-private.rs

```diff
@@ -0,0 +1,10 @@
+// check-pass
+// revisions: public private
+// [private]compile-flags: --document-private-items
+#![cfg_attr(private, deny(intra_doc_resolution_failure))]
+
+/// docs [DontDocMe]
```

Comment:
It sounds weird to me to have this warning when the flag `--document-private-items` is passed. In my opinion, it shouldn't complain in this case.

Reply:
It isn't warning when the flag is passed, look closer

Reply:
Oh you're right! My bad.